drivers/trigger/queries.go

```diff
@@ -133,12 +133,14 @@ func (d *Driver) getCreateTriggerSqlForOperation(listenerUid string, l *flash.Li
 
 	if operation != "TRUNCATE" {
 		statement += fmt.Sprintf(`
-			CREATE OR REPLACE TRIGGER "%s" AFTER %s ON %s FOR EACH ROW EXECUTE PROCEDURE "%s"."%s"();`,
-			triggerName, operation, d.sanitizeTableName(l.Table), d.Config.Schema, triggerFnName)
+			DROP TRIGGER IF EXISTS "%s" ON %s;
```

Comment:
Add comment

// Keep drop + create instead of 'create or replace' for Pgsql13 compatibility